README.md

```diff
@@ -0,0 +1,120 @@
+# Symbol Statistics Service
+
+It periodically collects information from the network's nodes and produces useful metrics and insights, such as the total number of nodes, their health or their geographical distribution.
+
+## Architecture
+
+The project uses Typescript, Node Express, and MongoDB as database.
+
+```text
+                       +---------------+
+                       |  Application  |
+                       +----+----^-----+
+                            |    |
+                            |    |
+                            |    |  Request / Response
+                            |    |
+ Symbol Statistics Service  |    |
++---------------------------v----+--+
+|                                   |
+| +-Express Server--------+         |
+| |                       |         |
+| |                       |         |
+| |     Task Manager      |         |
+| |                       |         |
+| |                       |         |
+| +-----------+-----------+         |
+|             |                     |
+| +-Services--v-----------------+   |
+| |                             |   |
+| | - Node Monitor              |   |
+| |                             |   |   +------------------+
+| | - Chain Height Monitor      +---+--->                  |
+| |                             |   |   | Mongodb Database |
+| | - Geolocation Monitor       <---+---+                  |
+| |                             |   |   +------------------+
+| +-----------------------------+   |
+|                                   |
++-----------------------------------+
+```
+
+### Node Monitor Task
+
+To discover nodes on the network, a request is made to the catapult-rest endpoint `/node/peer`. This returns the list of peers known to that node, which the task then explores recursively.
+
+To start the process, the task uses the nodes maintained by NEM Group.
```

Comment:
nem group?

Reply:
should change it to community

Reply:
I mean this sentence is incorrect, isn't it? which nodes are actually used now?